README.md

```diff
@@ -221,6 +247,10 @@ A response from the verifier endpoint (`/credentials/verify`) for a
 verifiable presentation might look like this (only HTTP response codes are
 checked):
 
+```http
+HTTP/1.1 200 OK
+```
+
 ```json
 {
   "checks": ["proof"],
```

Comment:
```suggestion
  "verified": true,
```

Reply:
https://github.com/digitalbazaar/vc/blob/9975a7b9184bb50b179566517f9d480a5d30170e/lib/index.js#L74-L88

I believe the verificationResult will resemble that provided it hasn't changed in the VC-API WG

Reply:
We're only documenting how the test suites do what they do. If/when we update to match the VC API, that'll be a different piece of work--as the test suite API is specifically a focused subset aimed at just test suite interfacing (vs. reinventing wheels).

Reply:
Similar to the above `statusResult` is not (any longer?) a thing in the VC API.

Reply:
What I can't see clearly here is what we actually respond with vs. what the VC API now wants/prefers.

We need to document what's implemented here...not what the VC API wants now...unless we refactor the API, but that's a heavy lift at this point with minimal value.

Thoughts?

Reply:
Going for shippable, and holding off on a full scrub of API use at this point. We can revisit if needed.